itou/utils/templatetags/nav.py

```diff
@@ -70,6 +70,16 @@ def __repr__(self):
         target=reverse("home:hp"),
         active_view_names=["dashboard:index"],
     ),
+    "employers-search": NavItem(
+        label="Un emploi inclusif",
+        target=reverse("search:employers_results"),
+        active_view_names=["search:employers_home", "search:employers_results"],
```

Comment:
Vu que c'est dans le même composant _tab_.
```suggestion
        active_view_names=["search:employers_home", "search:employers_results", "search:job_descriptions_results"],
```

Reply:
Bien vu, merci !